src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1181,15 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param stringInput  Parameter as a string
+     * @param prefix  Parameter prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String stringInput, String prefix) {
+    	String stringOutput = stringInput.substring(prefix.length());
```

Comment:
This code change is incorrect. This method will always remove the front part of the "stringInput" now. The intention of the method is to remove the prefix **if matched**. 

Hint: Google for replaceFirst().